src/main/java/edu/kis/vh/nursery/storage/IntArrayStack.java

```diff
@@ -1,11 +1,9 @@
-package edu.kis.vh.nursery;
-
-import edu.kis.vh.nursery.list.IntStorage;
+package edu.kis.vh.nursery.storage;
 
 public class IntArrayStack implements IntStorage{
 
     private static final int capacity = 12;
-    private static final int array_initial_index = -1;
+    private static final int array_initial_index = 0;
```

Comment:
ERROR: total variable initialization and validation should always remain -1.
The default return value should be moved to the interface (3.1.8).